.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - run: go vet ./...
   test:
     docker:
-    - image: circleci/golang:1.12
+    - image: circleci/golang:1.13
     steps:
     - checkout
     - restore_cache:
```

Comment:
see https://github.com/netdata/go.d.plugin/pull/162/files, there are some other places where we need to bump versions